telegram-translation-shortcuts.user.js

```diff
@@ -2,6 +2,7 @@
 // @name         Telegram Translation Platform Shortcuts
 // @namespace    https://github.com/jurf/telegram-translation-shortcuts
 // @description  Adds useful keyboard shortcuts to the Telegram Translation Platform
+// @author       Juraj Fiala
 // @include      https://translations.telegram.org/*
 // @version      0.3.2
```

Comment:
New version in a different commit?

Reply:
Yeah, otherwise people who already have it installed won’t get the update.

Reply:
no, I meant to ask if I should change the version number just before merge or in a separate commit after merge?